docs/security/threat_modeling.md

```diff
@@ -0,0 +1,69 @@
+# Threat Modeling
+
+Threat Modeling is (roughly) the discipline of imagining who might wish to attack you and what they want rather than starting from *how* they might do so. You can do both, of course.
+
+The [EFF's list of threat modeling questions](https://ssd.eff.org/en/module/your-security-plan) are a good starting point (link credit: @cotarg).
+
+## Ambient Network Malicious Actors
+
+Frequent, low-intensity attacks with no customization come from petty criminals, spam botnets and other actors with little or no specific interest in Beacon.
+
+In this case, the intent will be low-level compromise of the host machine. Normally this will occur via OS services rather than anything specific to Beacon itself.
+
+In general, Beacon is well-architected to defend this scenario. An outer static-site wrapper with only a single login URL is an excellent defense. As with nearly any application, "expose very little surface area" is the right approach.
+
+The accompanying operations strategy is to run as few additional services as possible and to make sure they aren't easily accessible from outside the host.
+
+Overall, this category is likely to be frequent but will not normally require human intervention. As a percentage of attacks, by far the largest mitigation is "don't have a WordPress login URL," followed closely by "don't run PHP."
```

Comment:
Can you please remove the reference to PHP? I don't want to have a jab at another programming language in the docs, even if it's coming from a security perspective.

Reply:
That's fair. I've removed it and revised the commit (not just added another one) in my branch to avoid having the same thing in the commit history.